eng/pipelines/templates/jobs/archetype-sdk-cmake-generate.yml

```diff
@@ -72,3 +78,64 @@ jobs:
           CmakeGeneratePath: sdk/${{ parameters.ServiceDirectory }}/${{ artifact.Path }}
           GenerateArgs: ${{ cmakeOption.Value }}
           Env: "$(CmakeEnvArg)"
+
+- job: CMakeSourceGenerate
```

Comment:
We don't need to duplicate this.  You should not need to update this file at all.  Just adding the option to the ci.yml should be enough